vulture_os/services/frontend/models.py

```diff
@@ -1172,6 +1172,11 @@ class Frontend(models.Model):
         help_text = _("ProofPoint TRAP API key"),
         default = "",
     )
+    proofpoint_trap_timeout = models.PositiveIntegerField(
```

Comment:
Please make this parameter 'generic' for every API collector: this seems like a change that can be implemented for all of them (you don't necessarily need to implement it in very collector yet, though)